src/SearchEngine.java

```diff
@@ -0,0 +1,111 @@
+import strategies.SearchStrategy;
+import strategies.StrategyALL;
+import strategies.StrategyANY;
+import strategies.StrategyNONE;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class SearchEngine {
```

Comment:
All your methods are `public`, please fix that.

Reply:
fixed